frontend/scss/components/templates/_default.scss

```diff
@@ -14,7 +14,7 @@
   margin-right: auto;
 
   @media (max-width: 767px) {
-    padding: 0 0 0 15px;
+    padding: 0 15px 0 15px;
```

Comment:
This could be shortened to `padding: 0 15px;` since vertical and horizontal values are the same.